src/crypto/index.ts

```diff
@@ -231,6 +231,14 @@ export enum CryptoEvent {
     KeyBackupStatus = "crypto.keyBackupStatus",
     KeyBackupFailed = "crypto.keyBackupFailed",
     KeyBackupSessionsRemaining = "crypto.keyBackupSessionsRemaining",
+    /**
+     * Fires when a new valid backup decryption key is in cache.
+     *
+     * The payload is the `KeyBackupInfo`.
```

Comment:
```suggestion
     * The payload is the {@link Crypto.KeyBackupInfo}.
```

(or whatever it is to make that link work right)

Reply:
Update that, the payload is now just the backup version (changed following your remark on when it's fired)